### ISSUE TITLE
[Feature Request] Allow horizontal bars

### ISSUE THREAD

**Issue by furstblumier** [lichie567/LMeter#82]
Would it be possible to have horizontal bar allignment similar to horizoverlay? I'm a big fan of your project, but that's the one thing I genuinely miss on linux ffxiv

**Comment by Cordil** [comment]
i would like that too, just copy horizo

**Comment by hoppyhopps** [comment]
I agree! It's a wonderful project, horizontal bars as a feature would make it even better.

Thanks for all your hard work.
